src/afl-fuzz-python.c

```diff
@@ -295,88 +295,94 @@ void deinit_py(void *py_mutator) {
 
 }
 
-void load_custom_mutator_py(afl_state_t *afl, char *module_name) {
+struct custom_mutator * load_custom_mutator_py(afl_state_t *afl, char *module_name) {
 
-  afl->mutator = ck_alloc(sizeof(struct custom_mutator));
-  afl->mutator->pre_save_buf = NULL;
-  afl->mutator->pre_save_size = 0;
+  struct custom_mutator * mutator;
 
-  afl->mutator->name = module_name;
+  mutator = ck_alloc(sizeof(struct custom_mutator));
+  mutator->pre_save_buf = NULL;
```

Comment:
Shouldn't this be part of the python mutator struct? (And also be cleare there?) It's not used anywhere else.